internal/cmd/completion/cmd.go

```diff
@@ -0,0 +1,71 @@
+package completion
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// Command creates the `completion` command
+func Command() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:   "completion [bash|zsh|fish|powershell]",
+		Short: "Generate completion script",
+		Long: `To load completions:
+
+Bash:
+
+  $ source <(saucectl completion bash)
```

Comment:
Is there other steps/packages requirement to make this work ? 

```
{14:25}~/Projects/tests ➭ bash
bash-5.1$ source <(../saucectl/saucectl completion bash)
bash-5.1$ saucectl bash: _get_comp_words_by_ref: command not found
demo-zsh.yml rbash: _get_comp_words_by_ref: command not found
bash: _get_comp_words_by_ref: command not found
^C
bash-5.1$
exit
{14:25}~/Projects/tests ➭
```

Reply:
By running `saucectl completion bash > /usr/local/etc/bash_completion.d/saucectl` it works out of the box.